_posts/2020-12-04-release-1.12.0.md

```diff
@@ -0,0 +1,332 @@
+ON o.currency = r.currency;
+```
+
+The previous example also shows how you can take advantage of the new `upsert-kafka` connector in the context of temporal table joins.
+
+**Hive Tables in Temporal Table Joins**
+
+You can also perform temporal table joins against Hive tables by either automatically reading the latest table partition as a temporal table ([FLINK-19644](https://issues.apache.org/jira/browse/FLINK-19644)) or the whole table as a bounded stream tracking the latest version at execution time. Refer to the [documentation]({{ site.DOCS_BASE_URL }}flink-docs-release-1.12/dev/table/connectors/hive/hive_read_write.html#temporal-table-join) for examples of using Hive tables in temporal table joins.
+
+<hr>
+
+### Other Improvements to the Table API/SQL
+
+**Kinesis Flink SQL Connector ([FLINK-18858](https://issues.apache.org/jira/browse/FLINK-18858))**
+
+From Flink 1.12, Amazon Kinesis Data Streams (KDS) is natively supported as a source/sink also in the Table API/SQL. The new Kinesis SQL connector ships with support for Enhanced Fan-Out (EFO) and Sink Partitioning. For a complete overview of supported features, configuration options and exposed metadata, check the [updated documentation]({{ site.DOCS_BASE_URL }}flink-docs-release-1.12/dev/table/connectors/kinesis.html).
+
+**Streaming Sink Compaction in the FileSystem/Hive Connector ([FLINK-19345](https://issues.apache.org/jira/browse/FLINK-19345))**
+
+Many bulk formats, such as Parquet, are most efficient when written as large files; this is a challenge when frequent checkpointing is enabled, as too many small files are created (and need to be rolled on checkpoint). In Flink 1.12, the file sink supports **file compaction**, allowing jobs to retain smaller checkpoint intervals without generating a large number of files. To enable file compaction, you can set `auto-compaction=true` in the properties of the FileSystem connector, as described in the [documentation]({{ site.DOCS_BASE_URL }}flink-docs-release-1.12/dev/table/connectors/filesystem.html#file-compaction).
+
+**Watermark Pushdown in the Kafka Connector ([FLINK-20041](https://issues.apache.org/jira/browse/FLINK-20041))**
+
+To ensure correctness when consuming from Kafka, it’s generally preferable to generate watermarks on a per-partition basis, since the out-of-orderness within a partition is usually lower than across all partitions. Flink will now push down watermark strategies to [emit **per-partition watermarks**]({{ site.DOCS_BASE_URL }}flink-docs-release-1.12/dev/table/connectors/kafka.html#source-per-partition-watermarks) from within the Kafka consumer. The output watermark of the source will be determined by the minimum watermark across the partitions it reads, leading to better (i.e. closer to real-time) watermarking. Watermark pushdown also lets you configure per-partition **idleness detection** to prevent idle partitions from holding back the event time progress of the entire application.
+
+**Newly Supported Formats**
```

Comment:
We also added a new changelog format for maxwell. The documentation for maxwell-json is under reviewing: https://github.com/apache/flink/pull/14245. Could you mention this new format in the table?